### PR TITLE
Hepatectomy

### PR DESCRIPTION
**Pull request by Moltijoe**
It's weird that all the other vital organs (brain, lungs, heart) have a one time surgery to fix them from being fucked.
But when it comes to the liver, replacement is the only option. This fixes that.
Surgery is done in the exact same way as the lobectomy.

just gotta make sure to stomach pump first, or the chemicals might re-kill the liver instantly

:cl:  
rscadd: Adds a new surgery to fix dead livers
/:cl:

### CODE REVIEW

File: code/modules/surgery/hepatectomy.dm [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+				/datum/surgery_step/mechanic_unwrench,
+				/datum/surgery_step/prepare_electronics,
+				/datum/surgery_step/hepatectomy,
+				/datum/surgery_step/mechanic_wrench,
+				/datum/surgery_step/mechanic_close)
+	requires_bodypart_type = BODYPART_ROBOTIC
+	lying_required = FALSE
+	self_operable = TRUE
+
+//lobectomy, removes the most damaged lung lobe with a 95% base success chance
```
Comment (ReddicusDragon) [inline code/modules/surgery/hepatectomy.dm:33]: update comment please

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
Update that comment then approved

File: code/modules/surgery/hepatectomy.dm (hunk 1 above)
Comment (Moltijoe) [reply code/modules/surgery/hepatectomy.dm:33]: no

Comment (ReddicusDragon) [reply code/modules/surgery/hepatectomy.dm:33]: Fair

**Review by ReddicusDragon — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**